src/components/VisualizationOptions/Options/DataIcon.js

```diff
@@ -0,0 +1,17 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import { CheckboxBaseOption } from './CheckboxBaseOption.js'
+
+const DataIcon = () => (
+    <CheckboxBaseOption
+        label={i18n.t('Show data item icon')}
+        helpText={i18n.t(
+            'If the data item has an icon, display it next to the value'
+        )}
+        option={{
+            name: 'icons',
```

Comment:
is `icons` coming from the backend? if not, maybe we can change it to something more descriptive like `showIcon`  (or even more detailed like `showDataItemIcon` if that makes sense) in case there are more icon related options coming in the future

Reply:
Yes, it is called `icons` in the backend.
I also thought maybe to use a more explicative name, but for the other options it seems we just keep the original name in the AO, which it makes sense for readability and debugging once you know that the names in the options correspond to the ones you see in the AO.